lib/provisioning/ProvisionDispatcher.js

```diff
@@ -58,36 +58,34 @@ class ProvisionDispatcher {
     addProvisions(provisionList, cb) {
         this._log.debug('add new provisions', { provisionList });
         async.series([
-            done => this._waitConnected(done),
-            done => this._populateZkHierarchy(done),
+            this._waitConnected,
```

Comment:
I think you need to `.bind(this)` in order to provide a closure rather than the bare class method